sxs/waveforms/alignment.py

```diff
@@ -549,3 +551,284 @@ def align4d(
     wa_prime = wa_prime.rotate(δSpin3)
 
     return optimum.cost, wa_prime, optimum
+
+def map_waveform_to_canonical_frame(wa, t_ref):
```

Comment:
I wonder if this could share could with [the stuff in `to_lvc_conventions`](https://github.com/sxs-collaboration/sxs/blob/7069e94f943e62ffe6fed4f54de708e24c29d6d0/sxs/waveforms/format_handlers/lvc.py#L105-L159).  Or at least, they should reference each other in a "See Also" section.